docs/products/kafka/kafka-connect/howto/iceberg-sink-connector.md

```diff
@@ -26,7 +26,7 @@ The connector supports the following catalog types:
 
 - [AWS Glue REST catalog](/docs/products/kafka/kafka-connect/howto/aws-glue-rest-catalog)
 - [AWS Glue catalog](/docs/products/kafka/kafka-connect/howto/aws-glue-catalog)
-
+- [JDBC catalog (PostgreSQL)](/docs/products/kafka/kafka-connect/howto/jdbc-catalog-postgres)
```

Comment:
Does this need to be changed to match the sidebar title?

Reply:
Good catch, updated to match the sidebar title.